@stellar/anchor-tests/src/schemas/sep38.ts

```diff
@@ -157,6 +157,12 @@ export const quoteSchema = {
     buy_amount: {
       type: "string",
     },
+    buy_delivery_method: {
```

Comment:
Does this need to be added to the price schema too?

Reply:
No, this is just for quotes

Reply:
Ah, I misread the protocol PR.